ctypesgen/printer_python/printer.py

```diff
@@ -425,6 +415,7 @@ def print_simple_macro(self, macro):
         # The macro translator makes heroic efforts but it occasionally fails.
         # We want to contain the failures as much as possible.
         # Hence the try statement.
+        # TODO(geisserml) Avoid try/except wrapper for simple constant definitions
```

Comment:
It looks cumbersome to exclude all such macros individually (if there are more than a handful?)

Would it be a good idea to **first start by logging a warning**  before `pass`, whenever we fail to handle a macro here ?

Reply:
Just realized macro exclude already exists (`--undefine` option).

I believe the macro translator rarely fails, so no try/except and individual excludes (or manual fixup) should be fine. Otherwise, I can also limit the change to integer constants for now and else keep the try/except.